source/CardGames.py

```diff
@@ -1,7 +1,30 @@
-
 import random
 import os
 
+#Displaying the rules for Go Fish!
+
```

Comment:
Great job on listing the rules. I have a question on why you did not just use one print statement?

Reply:
I felt like it looked more organized and easier to understand in a list. Listing the rules like how games usually have their rules rather than one big paragraph.